shared/native/notification-listeners.native.js

```diff
@@ -9,11 +9,5 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
   const fromShared = shared(dispatch, getState, notify)
   return {
     ...fromShared,
-    'keybase.1.NotifyApp.exit': () => { },
```

Comment:
don't register for this stuff to just throw it away, don't get it in the first place